spring-web/src/test/java/org/springframework/web/filter/RequestLoggingFilterTests.java

```diff
@@ -44,8 +44,96 @@
  */
 public class RequestLoggingFilterTests {
 
-	private final MyRequestLoggingFilter filter = new MyRequestLoggingFilter();
```

Comment:
Please revert the `@BeforeEach` change. It is unnecessary and undoes the exact opposite change introduce in https://github.com/spring-projects/spring-framework/commit/08ddd1b3bc783d0dc0ef1ab5e361315cbba90e10#diff-8c84ed40b500108bc760d43f76bbdaf9R43.